official/vision/beta/projects/yolo/modeling/heads/yolo_head.py

```diff
@@ -0,0 +1,121 @@
+# Lint as: python3
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import tensorflow as tf
+from yolo.modeling.layers import nn_blocks
+
+
+class YoloHead(tf.keras.layers.Layer):
+  """YOLO Prediction Head"""
+
+  def __init__(self,
+               min_level,
+               max_level,
+               classes=80,
+               boxes_per_level=3,
+               output_extras=0,
+               norm_momentum=0.99,
+               norm_epsilon=0.001,
+               kernel_initializer="glorot_uniform",
+               kernel_regularizer=None,
+               bias_regularizer=None,
+               activation=None,
+               **kwargs):
+    """
+    Yolo Prediciton Head initialization function.
+
+    Args:
+      min_level: `int`, the minimum backbone output level
+      max_level: `int`, the maximum backbone output level
+      classes: `int`, number of classes per category
+      boxes_per_level: `int`, number of boxes to predict per level
+      output_extras: `int`, number of additional output channels that the head
+        should predict for non-object detection and non-image classification
+        tasks
+      norm_momentum: `float`, normalization omentum for the moving average.
+      norm_epsilon: `float`, small float added to variance to avoid dividing by
```

Comment:
inconsistent end-of-line period or not

Reply:
Fixed for this PR. Need a more systematic way to add the periods at the end of the parameter docstrings